.github/ISSUE_TEMPLATE/release_testnet.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Testnet release
+about: Checklist for releasing a testnet
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Testnet Release
+
+Testnet name: X
+Release date: X
+Milestone link: X
+Testnet release manager: X
+
+# Testnet Release Manager Checklist
+
+Monday (week before release):
+- [ ] Create GitHub milestone, work with team to populate the milestone with tickets targeted for the release.
+
+Thursday:
+- [ ] Check in with team again in a release meeting and update the GitHub milestone to ensure it represents what will make it into the testnet.
+
+Friday: Prepare a PR (for merge on Monday) with the following:
+- [ ] Update `discord-addresses` file using the Galileo bot (`cargo run --release history --channel <channel>`)
+- [ ] Create a new `testnet/<moon>/allocations.csv` file describing the initial genesis allocations for this testnet (make sure all current team members and the Galileo bot have some assets!)
```

Comment:
This should be done on Monday, immediately following the release, so that the `testnet-preview` instances have a randomized version of the *next* chain id, rather than of the current one.

Reply:
Hmm, looks like this is still in the Friday (pre-release) section rather than the Monday (post-release) section ?

Reply:
so I moved the creation of the testnet directory and initial team allocations to Monday, but I left the step where we update the list of discord bot addresses (and adding allocations for them) on Friday (such that the faucet addresses posted during the week are included) - do we want to do them both on Monday?

Reply:
I think it's simpler to do them both on Monday -- it means there's a week-long lead time for faucet-to-genesis inclusion, but it's a simpler workflow, which seems like a win.